### PR TITLE
Update Lab env

### PR DESCRIPTION
**Pull request by katarinasupe**
### Description

Based on Toni's feedback, Lab ENV var has been updated and all ENVs ordered in alphabetical order.  Best practices on deploying Lab in K8s with Nginx Ingress service have been updated on the Kubernetes page.

### Pull request type

Please check what kind of PR this is:

- [x] Fix or improvement of an existing page

### Checklist:

- [x] Check all content with Grammarly
- [x] Perform a self-review of my code
- [x] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [x] The build passes locally
- [x] My changes generate no new warnings or errors
- [x] Add a corresponding label

### CODE REVIEW

**Review by kgolubic — APPROVED** [review]
Everything looks OK to me.

**Review by hal-eisen-MG — APPROVED** [review]
LGTM

**Comment by katarinasupe** [comment]
@kgolubic I didn't manage to incorporate Toni's suggestion

**Comment by kgolubic** [comment]
> @kgolubic I didn't manage to incorporate Toni's suggestion

I apologize. I'll fix this. I saw a `ship` label, and therefore I shipped it. My mistake.

File: pages/data-visualization/install-and-connect.mdx [hunk 1]
```diff
@@ -198,21 +198,22 @@ Configure Memgraph Lab using the following environment variables when running it
 
 | Variable                        | Description                                   | Type       | Default  |
 |---------------------------------|-----------------------------------------------|------------|----------|
-| `QUERY_MAX_LEN`                 | Max length of a Cypher query                  | `integer`  | `5000`   |
-| `MODULE_NAME_MAX_LEN`           | Max length of the query module name           | `integer`  | `1000`   |
+| `KEEP_ALIVE_TIMEOUT_MS`         | Max time during which Lab will hold the connection | `integer`  | `65000` |
```
[the remark this answers is not in the document]
Comment (kgolubic) [reply pages/data-visualization/install-and-connect.mdx:201]: Resolved as part of https://github.com/memgraph/documentation/pull/850.

**Pull request merged**